solutions/usaco-596.mdx

```diff
@@ -93,7 +93,62 @@ void ff(int i, int j)
 	}
 }
 ```
+## Alternate Faster Solution
 
```

Comment:
add time complexity here as well

Reply:
Done

Reply:
@nwatx shouldn't time complexity go under the explanation like [here](https://usaco.guide/general/adding-solution?lang=cpp#steps)?